src/server/http/__tests__/xsrf.js

```diff
@@ -29,10 +30,23 @@ describe('xsrf request filter', function () {
 
     await kbnServer.ready();
 
-    const routeMethods = nonDestructiveMethods.filter(method => method !== 'HEAD').concat(destructiveMethods);
     kbnServer.server.route({
       path: '/xsrf/test/route',
-      method: routeMethods,
+      method: nonDestructiveMethods.filter(method => method !== 'HEAD'),
```

Comment:
I can't see any reason to keep it as-is. It just makes the test-intention less clear.